cpp/test/IceSSL/configuration/SchannelTests.cpp

```diff
@@ -116,7 +116,8 @@ createServer(ServerAuthenticationOptions serverAuthenticationOptions, TestHelper
 Ice::CommunicatorPtr
 createClient(optional<ClientAuthenticationOptions> clientAuthenticationOptions = nullopt)
 {
-    return initialize(Ice::InitializationData{.clientAuthenticationOptions = clientAuthenticationOptions});
+    return initialize(Ice::InitializationData{
+        .clientAuthenticationOptions = clientAuthenticationOptions.value_or(ClientAuthenticationOptions{})});
```

Comment:
What's the difference between a nullopt ClientAuthenticationOptions and a default constructed ClientAuthenticationOptions?

This code clearly suggests there is a difference.

Reply:
The nullopt means to use the settings from IceSSL properties and its defaults.
Empty options mean using the system defaults, IceSSL properties are not considered.

Reply:
Then, I think it would be clearer for this `createClient` helper to use the same logic - no use nullopt as a substitute for empty ClientAuthenticationOptions.

And if the nullopt behavior is never acceptable, then make the parameter non-optional. Like:
```cpp
Ice::CommunicatorPtr
createClient(ClientAuthenticationOptions clientAuthenticationOptions)
```

Reply:
Fixed as suggested.